Ifeanyi/Chapter_2/exercise_2.14.5

```diff
@@ -0,0 +1,8 @@
+#When interest rate is 1%
+P = input('what is Principal amount: ')
+print(' interest rate is 1%')
+r = 1
+n = input('enter the number of times the intrest rate is compounded per year: ')
+t = input('enter the number of years: ')
+A=int(P)*(1+ float(int(r)/int(n)))**(int(n)*int(t))
```

Comment:
So make sure to write an actual script, well documented and make the letters human readable.